aten/src/ATen/xpu/CachingHostAllocator.cpp

```diff
@@ -36,6 +37,15 @@ struct XPUCachingHostAllocatorImpl
     // exit. Will be enabled once the issue is resolved.
     return false;
   }
+
```

Comment:
FYI @guangyey These are the only changes required to make XPU private memory pools work with pinned host memory, thanks to your suggestion on my first attempt at this. I hope it looks good to you.